src/features/HomePage/components/NewestMentors.tsx

```diff
@@ -0,0 +1,117 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Mentor,
+  selectFilteredMentors,
+  useGetMentorsQuery,
+} from '@/features/MentorPage/mentorPageApi';
+import { useAppSelector } from '@/store';
+
+import ListCard from '@/features/MentorPage/components/MentorList/MentorCard/List';
+import MentorCard from '@/features/MentorPage/components/MentorList/MentorCard/Expanded';
+import { OUTER_HORIZONTAL_MARGIN, palette } from '@/components/variables';
+import Text from '@/components/Text';
+import { TextButton } from '@/components/Buttons';
+
+const NewestMentors = () => {
+  const { t } = useTranslation('home');
+  const navigate = useNavigate();
+  const navigateToMentors = () => navigate('/mentors');
+
+  const { isLoading } = useGetMentorsQuery();
+  const [selectedMentor, setSelectedMentor] = useState<Mentor | null>(null);
+  const mentorsToShow = useAppSelector(selectFilteredMentors()).slice(0, 2);
```

Comment:
Did not notice this before. I think we should introduce a selector that will 
a) sort mentors by created-date
b) give only n, amounts of mentors (n being a argument for selector)

Reply:
Ah yes, good catch! I seem to have overlooked this amidst all the changes happening here!

Reply:
I made a placeholder selector for this, but maybe we'll finish it in another ticket.

Reply:
Now done! 💪